bot/utility/utility.py

```diff
@@ -95,6 +96,34 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.command(name='howto')
+    @command_log
+    async def howto(self, ctx: commands.Context, what: Optional[str] = None):
+        """Handler for the `howto code` command.
+
+        Explains how to do various things
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            what (Optional[str]): What the user wants to know about
+        """
+        subcommands = {
+            "code": ("How to format code with discord markdown", howto_handler.code),
+        }
+        if what in subcommands:
+            embed = subcommands[what][1]()
+        else:
+            if what is not None:
```

Comment:
Can be reduced to `if what`, assuming that this will only be a string and you only want to enter if it has any non-empty value.